app/utils/credentials.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const { InvalidConfigurationError } = require('../errors')
+
+const CREDENTIAL_STATE = {
+  CREATED: 'CREATED',
+  ENTERED: 'ENTERED',
+  VALIDATED: 'VALIDATED_WITH_LIVE_PAYMENT',
```

Comment:
This should be `VERIFIED_WITH_LIVE_PAYMENT` https://github.com/alphagov/pay-connector/blob/master/src/main/java/uk/gov/pay/connector/gatewayaccountcredentials/model/GatewayAccountCredentialState.java